src/domain/kaivuilmoitus/Contacts.tsx

```diff
@@ -28,12 +52,51 @@ export default function Contacts() {
   const postalAddressRequired = !ovt || !invoicingOperator;
   const ovtRequired = !ovtDisabled && (!streetName || !postalCode || !city);
 
+  const isMountedForSelectedContactType = useRef(false);
+  const isMountedForRegistryKey = useRef(false);
```

Comment:
Are both of these isMounted variables needed? Are there some problems if using only one in both useEffects?

Reply:
I use the same technique as in ApplicationContacts: a single isMounted that is set in the last useEffect. Also added the same implementation for reverting back to the original hidden value.